export/client/registration.go

```diff
@@ -109,6 +109,12 @@ func addReg(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if reg.Name == "" {
```

Comment:
Could you move this logic to registration.Validate()? And add/modify tests following the new logic

Reply:
Name can't be empty when creating a registration but it can be empty when updating? Why can it be empty in that case?

Reply:
When updating you can send something like this:
`{"id":"57db5bd2add4d779d38ff066","compression":"GZIP"}`
So, there is no need to send the name

Reply:
You should not call validate on update. It will set some of the missing fields to the default values( ompNone for compression) and will test that the remaining fields had a valid value.

Reply:
@feclare After recheck the code, I realized we can move the name check to registration.Validate() without more modifications. Also I have had to modify registration_test.go

Reply:
Yes, you are right. After inspecting the code again the validate when updating a registration also make sense.